cmd/frontend/internal/markdown/markdown.go

```diff
@@ -23,9 +32,41 @@ func Render(content string) string {
 		policy.AllowAttrs("aria-hidden").Matching(regexp.MustCompile(`^true$`)).OnElements("a")
 		policy.AllowAttrs("type").Matching(regexp.MustCompile(`^checkbox$`)).OnElements("input")
 		policy.AllowAttrs("checked", "disabled").Matching(regexp.MustCompile(`^$`)).OnElements("input")
-		policy.AllowAttrs("class").Matching(regexp.MustCompile("^language-[a-zA-Z0-9]+$")).OnElements("code")
+		policy.AllowAttrs("class").Matching(regexp.MustCompile("^(?:chroma-[a-zA-Z0-9\\-]+)|chroma$")).OnElements("pre", "code", "span")
+
+		origTypes := chroma.StandardTypes
+		sourcegraphTypes := map[chroma.TokenType]string{}
+		for k, v := range origTypes {
+			if k == chroma.PreWrapper {
+				sourcegraphTypes[k] = v
+			} else {
+				sourcegraphTypes[k] = fmt.Sprintf("chroma-%s", v)
+			}
+		}
+		chroma.StandardTypes = sourcegraphTypes
+
+		renderer = goldmark.New(
+			goldmark.WithExtensions(
+				extension.GFM,
+				highlighting.NewHighlighting(
+					highlighting.WithFormatOptions(
+						chromahtml.WithClasses(true),
+						chromahtml.WithLineNumbers(false),
+					),
+				),
+			),
+			goldmark.WithParserOptions(
+				parser.WithAutoHeadingID(),
+			),
+			goldmark.WithRendererOptions(
+				html.WithUnsafe(),
+			),
+		)
 	})
 
-	unsafeHTML := gfm.Markdown([]byte(content))
-	return string(policy.SanitizeBytes(unsafeHTML))
+	var buf bytes.Buffer
+	if err := renderer.Convert([]byte(content), &buf); err != nil {
+		return "", err
+	}
+	return string(policy.SanitizeBytes(buf.Bytes())), nil
```

Comment:
nit: we can skip some string/bytes conversions by using `strings.Builder`

```suggestion
	var buf strings.Builder
	if err := renderer.Convert([]byte(content), &buf); err != nil {
		return "", err
	}
	return policy.Sanitize(buf.String()), nil
```